app/graphql/types/query_type.rb

```diff
@@ -2,16 +2,24 @@
 
 module Types
   class QueryType < Types::BaseObject
-    # Add root-level fields here.
-    # They will be entry points for queries on your schema.
+    # # Add root-level fields here.
+    # # They will be entry points for queries on your schema.
+    #
+    # field :hello_world, # name of method
+    #       String, # Type the data will be
+    #       null: false, # If it can ever return a null value
+    #       description: 'An example field added for your reference!' # Optional Description
+    #
+    # def hello_world
+    #   'Hello World - This is your Interview Challenge!\nGood luck.'
+    # end
 
-    field :hello_world, # name of method
-          String, # Type the data will be
-          null: false, # If it can ever return a null value
-          description: 'An example field added for your reference!' # Optional Description
+    field :getAllCandidates, [Types::CandidateType], null: false do
+      description "This will get all the candidates."
+    end
 
-    def hello_world
-      'Hello World - This is your Interview Challenge!\nGood luck.'
+    def getAllCandidates
```

Comment:
Added this method with an ActiveRecord query for retrieving all of the candidates.